Singularity/Singularity/Input/InputManager2.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics.PackedVector;
+using Microsoft.Xna.Framework.Input;
+
+namespace Singularity.Input
+{
+    static class InputManager2
```

Comment:
how come we have a second inputManager?

Reply:
Temporary. The input manager that exists didn't do what we needed to do so I made a new input manager. This will be removed in a future version and merged with the real input manager

Reply:
why not directly improve the inputManager?

Reply:
Didn't want to touch the real inputmanager yet. Also during the last weekly scrum Julian and I discussed improving my implementation (which is a quick and dirty implementation) and putting it together with the real inputmanager next week